http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/DataprocConfig.scala

```diff
@@ -7,6 +7,7 @@ final case class DataprocConfig(
   applicationName: String,
   dataprocDefaultRegion: String,
   dataprocZone: Option[String],
+  //TODO: refactor a common config with "realm" in swagger config and the field in pubsub config
```

Comment:
What field(s) is this comment referring to?

Reply:
the leoGoogleProject field. We have googleProject being read into 3 different config objects currently.

Search `"{{$googleProj}}"` here to confirm it is indeed the same exact value, templated in 3 different times in 3 different places.

https://github.com/broadinstitute/firecloud-develop/blob/dev/base-configs/leonardo/leonardo.conf.ctmpl

Reply:
Ah I see, yeah